src/whylogs/app/config.py

```diff
@@ -221,8 +221,6 @@ def __init__(
         self.pipeline = pipeline
         self.verbose = verbose
         self.writers = writers
-        if not metadata:
```

Comment:
Can't comment on unchanged line. But can you update line 214 as well:

```        
metadata: Optional[MetadataConfig] = False,
```

should be
```
        metadata: Optional[MetadataConfig] = None,
```